src/client/eppo-client.ts

```diff
@@ -866,12 +866,14 @@ export default class EppoClient {
    *
    * @param subjectKey an identifier of the experiment subject, for example a user ID.
    * @param subjectAttributes optional attributes associated with the subject, for example name and email.
-   * @param banditActions
+   * @param banditActions optional attributes associated with the bandit actions
+   * @param salt a salt to use for obfuscation
    */
   getPrecomputedConfiguration(
     subjectKey: string,
     subjectAttributes: Attributes | ContextAttributes = {},
     banditActions: Record<FlagKey, BanditActions> = {},
+    salt = '',
```

Comment:
So a couple ways to use this:

1. Monkey patch this during `init` in the node server sdk

```
  clientInstance.getPrecomputedConfiguration = (
    subjectKey: string,
    subjectAttributes: Attributes | ContextAttributes = {},
    banditActions: Record<FlagKey, BanditActions> = {},
    salt: string = generateSalt(),
  ) => {
    return clientInstance.getPrecomputedConfiguration(
      subjectKey,
      subjectAttributes,
      banditActions,
      salt,
    );
  };
```

This means we'd allow the end-user to supply their own `salt` if they desire the obfuscation output to be consistent.

2. Or if we want to change the function signature, then we do something like the js client sdk and export a new class `export class EppoNodeClient extends EppoClient {`